force-app/main/default/classes/TwilioSendSMS.cls

```diff
@@ -0,0 +1,26 @@
+public class TwilioSendSMS {
+    
+    public static String sendSMS(String smsBody, String toNumber){
```

Comment:
@smaheshwari1598 You can return HTTPResponse to caller then caller service will have more information regaring error message
```suggestion
    public static String sendSMS(String smsBody, String toNumber){
```